Composer/packages/client/src/styles.ts

```diff
@@ -81,7 +81,18 @@ export const bot = css`
 `;
 
 export const botButton = css`
-  margin-left: 15px;
+  margin-left: 5px;
+`;
+
+export const errorButton = css`
+  color: #a80000;
```

Comment:
Can we use a value that is exported from the fabric package? Like `SharedColors` or something?